src/levanter/models/whisper.py

```diff
@@ -60,19 +61,30 @@ class WhisperConfig(HFCompatConfig):
     gradient_checkpointing: bool = True
 
     # Attention-related config
-    upcast_attn: bool = False
+    upcast_attn: bool = True
     use_flash_attention: bool = False
     flash_attention_block_size: Optional[int] = None
 
+    # Tokenizer Info
+    # (Annoyingly Required for HF Compat because no separation of concerns)
```

Comment:
I could probably work this into hf_compat at export time since we have both the model and the tokenizer there at that stage. That way the Levanter model keeps its separation of concerns, but export still goes smoothly to HF.

The HF Whisper implementation requires this defined in their model config because you can call `model.generate` WITHOUT a tokenizer (so it uses these to force the BOS token and stop generation).

Reply:
I'll see if I can make that work cleanly before I merge.

Reply:
Relevant change here: https://github.com/stanford-crfm/levanter/pull/522/files#diff-4ad20b164ff0ec1d287ec49989d2861baf52b206289753541840ba9f448f8fc9R671-R687